src/utils/StateUtils.ts

```diff
@@ -0,0 +1,16 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ext } from '../extensionVariables';
+
+export async function getFromWorkspaceState<T>(key: string): Promise<T> {
```

Comment:
do we think having a util class for workplace momentos is needed? I thought it would be nice to have a util class that handles it, but we can go either way.

Reply:
Maybe overkill since it's a one-liner that replaces a one-liner, but I don't feel strongly one way or the other.